python-checks/src/main/java/org/sonar/python/checks/IgnoredParameterCheck.java

```diff
@@ -61,11 +67,39 @@ public void initialize(Context context) {
           // no usages in unreachable blocks
           .filter(assignment -> !isSymbolUsedInUnreachableBlocks(lva, unreachableBlocks, assignment.symbol))
           .filter((assignment -> !isUsedInSubFunction(assignment.symbol, functionDef)))
-          .forEach(assignment -> ctx.addIssue(assignment.element, String.format(MESSAGE_TEMPLATE, assignment.symbol.name())));
+          .forEach(assignment -> {
+            var issue = ctx.addIssue(assignment.element, String.format(MESSAGE_TEMPLATE, assignment.symbol.name()));
+            assignment.symbol.usages().stream()
+              .filter(Usage::isBindingUsage)
+              .filter(u -> u.kind() != Usage.Kind.PARAMETER)
+              .map(Usage::tree)
+              .collect(groupAssignmentByParentStatementList())
+              .values()
+              .stream()
+              .sorted(Comparator.comparing(t -> t.firstToken().line()))
+              .map(IgnoredParameterCheck::mapToParentAssignmentStatementOrExpression)
+              .filter(Objects::nonNull)
```

Comment:
I think we can remove this now, since `mapToParentAssignmentStatementOrExpression` is guaranteed to return a non-null value.

Reply:
done